analyzers/src/SonarAnalyzer.CSharp/Rules/SillyBitwiseOperation.cs

```diff
@@ -78,8 +78,10 @@ private void CheckBinary(SyntaxNodeAnalysisContext context, int constValueToLook
         }
 
         private static bool IsFieldOrProperty(SemanticModel semanticModel, SyntaxNode node) =>
-            semanticModel.GetSymbolInfo(node).Symbol is { } symbol
-            && symbol.ContainingNamespace.Name != "System"  // Excluding built-in constants
-            && symbol.Kind is SymbolKind.Field or SymbolKind.Property;
+            semanticModel.GetSymbolInfo(node).Symbol is { Kind: SymbolKind.Field or SymbolKind.Property } symbol
+            && !FieldOrPropertyBelongToSystemNamespace(symbol);
```

Comment:
I think we should move the `FieldOrPropertyBelongToSystemNamespace` call to `FindConstant` since it's a bit misleading. Even if it's in the system namespace it is still a field or property.

Reply:
Done